definitions/json/CaseEvent.json

```diff
@@ -805,13 +805,12 @@
     "ID": "viewAllNotifications",
     "Name": "View notifications",
     "Description": "View all judgement, orders and notifications",
-    "DisplayOrder": 99,
     "PreConditionState(s)": "*",
     "PostConditionState": "*",
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "N",
-    "EventEnablingCondition": "",
+    "EventEnablingCondition": "caseType=\"dummy\"",
```

Comment:
Why use dummy here?

Reply:
To remove it from "Next step" drop down list